helpers/helpers_emba_dependency_check.sh

```diff
@@ -194,8 +194,46 @@ setup_nikto() {
   fi
 }
 
+check_emba_version(){
+  LATEST_EMBA_VERSION="$(echo "${LATEST_EMBA_VERSION}" | cut -d "-" -f1)"
+  if [[ "$(printf '%s\n' "${LATEST_EMBA_VERSION}" "${EMBA_VERSION}" | sort -V | head -n1)" = "${LATEST_EMBA_VERSION}" ]]; then
+    echo -e "    EMBA version - ${GREEN}ok${NC}"
+  else
+    echo -e "    EMBA version - ${ORANGE}Updates available${NC}"
+  fi
+}
+
+check_git_hash(){
+  local REMOTE_HASH=""
+  local LOCAL_HASH=""
+  if git rev-parse --is-inside-work-tree >/dev/null 2>&1 ; then
+    REMOTE_HASH="$(curl --connect-timeout 5 -s -o - https://github.com/e-m-b-a/emba | grep "spoofed_commit_check" | sed -E 's/.*commit_check\/([a-zA-Z0-9]{8}).*/\1/')"
+    LOCAL_HASH="$(cut -d "-" -f2 ./config/VERSION.txt)"
+
+    if [[ "${REMOTE_HASH}" == "${LOCAL_HASH}" ]]; then
+      echo -e "    EMBA github version - ${GREEN}ok${NC}"
+    else
+      echo -e "    EMBA github version - ${ORANGE}Updates available${NC}"
+    fi
+  fi
+}
+
+check_docker_image(){
+  local LOCAL_DOCKER_HASH=""
+  local REMOTE_DOCKER_HASH=""
+  LOCAL_DOCKER_HASH="$(sudo docker image inspect embeddedanalyzer/emba:latest --format '{{json .RepoDigests}}' | jq . | grep "sha" | sed -E 's/.*sha256:([0-9|[a-z]+)"/\1/' )"
+  REMOTE_DOCKER_HASH="$(sudo docker manifest inspect embeddedanalyzer/emba:latest -v | jq . | grep "digest" | head -n1 | awk '{print $2}' | sed -E 's/"sha256:(.+)",/\1/')"
```

Comment:
I think we need todo a || true at the end. Otherwise we are probably running into issues in strict mode. Have you tested with -S?

Reply:
I tested it and I did not see anything weird, but I will add it anyways.